src/Core/NuGet/Core/Package.nuspec

```diff
@@ -24,14 +24,40 @@
       </group>
       <group targetFramework="net46">
       </group>
+      <group targetFramework=".NETStandard,Version=v1.5">
```

Comment:
So you changed to NetStandard now. @davidfowl - this is the main question. Should we use netstandard or netstandardAPP here?